pkg/api/content_units.go

```diff
@@ -1,16 +1,16 @@
 package api
 
 type ContentUnitListRequest struct {
-	UUID   string `param:"uuid"`    // Identifier of the repository
-	Search string `query:"search"`  // Search string based query to optionally filter-on
-	SortBy string `query:"sort_by"` // SortBy sets the sort order of the result
+	UUID   string `param:"uuid" validate:"required"`    // Identifier of the repository
+	Search string `query:"search" validate:"required"`  // Search string based query to optionally filter-on
+	SortBy string `query:"sort_by" validate:"required"` // SortBy sets the sort order of the result
 }
 
 type ContentUnitSearchRequest struct {
-	URLs   []string `json:"urls,omitempty"`  // URLs of repositories to search
-	UUIDs  []string `json:"uuids,omitempty"` // List of repository UUIDs to search
-	Search string   `json:"search"`          // Search string to search content unit names
-	Limit  *int     `json:"limit,omitempty"` // Maximum number of records to return for the search
+	URLs   []string `json:"urls,omitempty"`             // URLs of repositories to search
+	UUIDs  []string `json:"uuids,omitempty"`            // List of repository UUIDs to search
+	Search string   `json:"search" validate:"required"` // Search string to search content unit names
+	Limit  *int     `json:"limit,omitempty"`            // Maximum number of records to return for the search
 }
```

Comment:
The required inputs for ContentUnitSearchRequest are URLs and UUIDs. With that said, only one of them is required. So I'm not sure if it makes sense to mark them both as required, or neither?

In theory we allow an empty search, so I wouldn't say search is required